resources/js/components/studentProfile/StudentPhoto.vue

```diff
@@ -0,0 +1,24 @@
+<template>
+	<div class="row justify-content-end py-3 px-2">
```

Comment:
You don't need this div or the next one. Just move the "py-3 px-2" to this element's parent instead. 👍